### PR TITLE
k8s 1.14.3 release

### PR DESCRIPTION
**Pull request by MarcelMue**
towards https://github.com/giantswarm/giantswarm/issues/6109

### CODE REVIEW

**Comment by mazzy89** [comment]
@MarcelMue have you checked that no changes have been introduced in the new manifests for calico `3.7.x`?

**Comment by MarcelMue** [comment]
Not yet.

**Comment by mazzy89** [comment]
Ok then please check that the manifests we have don't diverge with the new manifests because we run those ones. I've checked the changelong.

File: v_4_4_0/files/k8s-resource/calico-all.yaml [hunk 1]
```diff
@@ -120,11 +120,14 @@ spec:
         kubernetes.io/os: linux
       hostNetwork: true
       tolerations:
+        # Make sure calico-node gets scheduled on all nodes.
```
Comment (MarcelMue) [inline v_4_4_0/files/k8s-resource/calico-all.yaml]: These tolerations were changed upstream, not sure if it makes a big difference.

File: v_4_4_0/files/k8s-resource/calico-all.yaml [hunk 2]
```diff
@@ -501,6 +505,8 @@ rules:
     verbs:
       # Needed for clearing NodeNetworkUnavailable flag.
       - patch
+      # Calico stores some configuration information in node annotations.
```
Comment (MarcelMue) [inline v_4_4_0/files/k8s-resource/calico-all.yaml]: There are a few more RBAC changes but it seems related to TYPHA which we don't use IIRC

File: v_4_4_0/files/k8s-resource/calico-all.yaml [hunk 3]
```diff
@@ -420,6 +423,7 @@ spec:
         - name: etcd-certs
           hostPath:
             path: /etc/kubernetes/ssl/etcd
+            defaultMode: 0400
```
Comment (MarcelMue) [inline v_4_4_0/files/k8s-resource/calico-all.yaml]: Upstream uses a secret for this, trying to change the mode with our mounted file as well :man_shrugging:

**Comment by mazzy89** [comment]
I've noticed that `ConfigMap` has this property:

```
  # Typha is disabled.
  typha_service_name: "none"
```

I would add it to the config. WDYT?

**Comment by mazzy89** [comment]
Tested and seems all good. @MarcelMue we can go ahead

**Comment by mazzy89** [comment]
I'm going to merge it and then in a different PR update the changelog and create a new WIP @MarcelMue

**Comment by MarcelMue** [comment]
Ack - let's go @mazzy89  :+1:

**Pull request merged**